content/en/security/application_security/setup/envoy-gateway.md

```diff
@@ -0,0 +1,254 @@
+---
+title: Enabling App and API Protection for Envoy Gateway
+code_lang: envoy-gateway
+code_lang_weight: 50
+further_reading:
+  - link: 'https://github.com/DataDog/dd-trace-go/tree/main/contrib/envoyproxy/go-control-plane/cmd/serviceextensions'
+    tag: "Source Code"
+    text: "Envoy integration's source code"
+  - link: "/security/default_rules/?category=cat-application-security"
+    tag: "Documentation"
+    text: "OOTB App and API Protection Rules"
+  - link: "/security/application_security/troubleshooting"
+    tag: "Documentation"
+    text: "Troubleshooting App and API Protection"
+---
+
+{{< callout url="#" btn_hidden="true" header="App and API Protection for Envoy Gateway is in Preview" >}}
+To try the preview of App and API Protection for Envoy Gateway, use the following setup instructions.
+{{< /callout >}}
+
+You can enable Datadog App and API Protection for traffic managed by [Envoy Gateway][1]. The Datadog Envoy Gateway integration allows Datadog to inspect and protect your traffic for threat detection and blocking directly at the edge of your infrastructure.
+
+## Prerequisites
+
+1. A running Kubernetes cluster with [Envoy Gateway][1] installed.
+2. The [Datadog Agent is installed and configured][2] in your Kubernetes cluster.
+    - Ensure [Remote Configuration][3] is enabled and configured to enable blocking attackers through the Datadog UI.
+    - Ensure [APM is enabled][4] in the Agent. *This allows the external processor service to send its own traces to the Agent.*
+      - Optionally, enable the [Cluster Agent Admission Controller][5] to automatically inject the Datadog Agent host information to the App and API Protection External Processor service.
+
+## Enabling threat detection
+
+Enabling App and API Protection with Envoy Gateway involves two steps:
+
+1. Deploying the Datadog External Processor service in your cluster.
+2. Configuring an `EnvoyExtensionPolicy` that points to the processor service to direct traffic from your Envoy Gateway to this service.
```

Comment:
```suggestion
2. Configure an `EnvoyExtensionPolicy` that points to the processor service. This will direct traffic from your Envoy Gateway to this service.
```